timescaledb/contribute-to-docs.md

```diff
@@ -86,16 +86,16 @@ that case, feel free to ignore them.
 *Adverbs*
 : Do not use.
 
-:+1: Install TimescaleDB.
+&#128077; Install TimescaleDB.
```

Comment:
@jacobprall What's best practice here? I thought we discussed using the HTML codes previously and Gatsby broke them, but they seem to be working now?

Reply:
Happy to do these any way we agree on but the Github-style ones don't currently display as emojis :) Unless the decision was that we'd prefer to leave them as plaintext?

Reply:
Interesting. Not seeing any errors in the logs for this deploy, and they appear to be working on staging - no reason to think they wouldn't work in prod. I'd say the html codes are the way to go - but I will keep an eye on deploys to make sure they don't break any deploys

Reply:
Thanks Jacob.